Makefile

```diff
@@ -36,6 +36,9 @@ stop: ## Stop the container
 		docker rm --force ${EXISTING_CONTAINERS};\
 	fi
 
+rebuild: ## Stop, build and restart the container
+	make stop build restart
```

Comment:
Wonder about having commands for each environment instead, that do the `stop build start watch` dance (eg `make dev` and `make prod`).

`stop` and `build` can happen in parallel, so something like:

```make
dev:
	ENV=dev make --jobs=2 stop build
	ENV=dev make start watch
```